core/plugins/org.polarsys.capella.core.data.migration/src/org/polarsys/capella/core/data/migration/contribution/ActivateDiagramFiltersContribution.java

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+ * Copyright (c) 2022 THALES GLOBAL SERVICES.
```

Comment:
Copyright must be changed before merge.

Copyright (c) 2022 Obeo

* Contributors:
 *    Obeo - initial API and implementation
 
 (or nlepin, or funder of the change, + bug ID or intitial implementation)